docs/python/examples.ipynb

```diff
@@ -1686,7 +1686,418 @@
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After all the basins are set. The component inbetween the basins needs to be determined. For polder basin 5 (node 12), it needs to be maintain at level 0.0 meter. This means that either there should be no water in this basin, or the basin bottom is lower than sea level, and the water level should be maintained at sea level. \n",
+    "\n",
+    "If it is the first case, a pump is needed to make sure there is no water left in the basin. If it is the second case, a normal outlet with minimum crest level of 0.0 is needed. In this example, the first case is assumed. Therefore, a pump is set."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Setup the pumps:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model.pump.add(\n",
+    "    Node(13, Point(4.0, -1.0)),\n",
+    "    [pump.Static(flow_rate=[0.5 / 3600])],\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "According to the description of situation 1 and 2, the water in one polder basin need to be able to flow to the downstream basin if the current basin has too much water that is higher than the setpoint or if the downstream basin requires more water. The functionality of the outlet with a minimum crest satisfy this feature. Take polder basin 1 (node 4) as an example. If the downstream polder has water shortage, polder basin 1 need to flow water as long as the water level is above 1.95 meter. So the minimum crest level for the outlet would be 1.95 meter."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Setup the outlets:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set up outlet\n",
+    "model.outlet.add(\n",
+    "    Node(2, Point(0.0, -1.0)),\n",
+    "    [outlet.Static(flow_rate=[2 * 0.5 / 3600], min_crest_level=[0.0])],\n",
+    ")\n",
+    "model.outlet.add(\n",
+    "    Node(5, Point(0.0, -3.0)),\n",
+    "    [outlet.Static(flow_rate=[0.5 / 3600], min_crest_level=[1.95])],\n",
+    ")\n",
+    "model.outlet.add(\n",
+    "    Node(7, Point(1.0, -4.0)),\n",
+    "    [outlet.Static(flow_rate=[0.5 / 3600], min_crest_level=[1.45])],\n",
+    ")\n",
+    "model.outlet.add(\n",
+    "    Node(9, Point(3.0, -4.0)),\n",
+    "    [outlet.Static(flow_rate=[0.5 / 3600], min_crest_level=[0.95])],\n",
+    ")\n",
+    "model.outlet.add(\n",
+    "    Node(11, Point(4.0, -3.0)),\n",
+    "    [outlet.Static(flow_rate=[0.5 / 3600], min_crest_level=[0.45])],\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "However, to maintain the water level to be at the setpoint, PID controllers are needed. The targets of the PID controllers are set to the setpoints"
```

Comment:
```suggestion
    "When using Outlets as connecting nodes, the flow over the Outlet needs to be controlled to maintain the water level at the setpoint. For this purpose we introduce local PidControllers, where the targets of the PidControllers are set to the setpoints. Disadvantage of this local control approach is the delay that is introduced to transport the 'basin X has a shortage' message upstream through the cascade to the inlet. Current functionality does not offer the capability for PidControl to take multiple observations into account when controlling the inlet. Combining multiple observations in one control is feasible with DiscreteControl. This could be an alternative approach to controlling the inlet for the cascading water system."
```